.ci/package.json

```diff
@@ -4,15 +4,18 @@
   "description": "Continuous integration of Zotero translators",
   "main": "index.js",
   "scripts": {
-    "test": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 -n1 -P8 ./checkTranslator.sh --skip-warn",
-    "start": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 -n1 -P1 ./checkTranslator.sh"
+    "errors": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 eslint -c eslintrc --quiet",
```

Comment:
1) Add spaces around `|`, here and below
2) Why not move `eslintrc` to `.eslintrc` and have it be implicit?

Reply:
1. Done, but I took that from `checkTranslators.sh`
2. eslint looks in the directory where `.git` lives so it won't find .eslintrc there